python/mxnet/gluon/block.py

```diff
@@ -1040,7 +1043,9 @@ def register_child(self, block, name=None):
         super(HybridBlock, self).register_child(block, name)
         self._clear_cached_op()
 
-    def hybridize(self, active=True, **kwargs):
+    def hybridize(self, active=True, backend='', optargs={}, **kwargs):
```

Comment:
maybe `optargs` should be `backend_args` for consistency? we will continue to pass kwargs to the C++ CachedOp instance. So want to disambiguate against options that will be used for partitioning.

Reply:
please set default to `None` instead of `''`